Bottom: 8ab05755f359a7795fcb7a5a793866cad03d1855
Top:    8e1e555675ee4f1ed8e6a0ecaf14dcd53dfc3b04
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 18:14:30 +0000

Refresh of flags-c-avoid-compiler-warning

---

diff --git a/src/flags.c b/src/flags.c
index c33cfb6..fdaef44 100644
--- a/src/flags.c
+++ b/src/flags.c
@@ -163,7 +163,7 @@ FlagLayerActive (int n)
   return current_layer == n;
 }
 
-#define OffsetOf(a,b) (int)(&(((a *)0)->b))
+#define OffsetOf(a,b) (int)(size_t)(&(((a *)0)->b))
 
 HID_Flag flags_flag_list[] = {
   {"style", FlagCurrentStyle, 0},
